code/_onclick/telekinesis.dm

```diff
@@ -39,7 +39,7 @@
 /obj/item/attack_tk(mob/user)
 	if(user.stat || !isturf(loc))
 		return
-	if(user.dna?.GetSEState(GLOB.teleblock) && !user.get_active_hand()) // both should already be true to get here
+	if(user.dna && HAS_TRAIT(user, TRAIT_TELEKINESIS) && !user.get_active_hand()) // both should already be true to get here
```

Comment:
Now that I think about it, I don't think you need these `if(X.dna` checks anymore. Since before they were needed to access the `GetSEState` proc. Personally, I say remove them.

Reply:
removed the checks